src/containers/costume-tab.jsx

```diff
@@ -224,7 +224,8 @@ class CostumeTab extends React.Component {
                 ]}
                 items={costumeData}
                 selectedItemIndex={this.state.selectedCostumeIndex}
-                onDeleteClick={target.costumes.length > 1 ? this.handleDeleteCostume : null}
+                onDeleteClick={target && target.costumes && target.costumes.length > 1 ?
```

Comment:
`target` can't be falsy, there is an early exit, also `costumeData` seems to built from `(target.costumes || [])` so you could just say `costumeData.length > 1` ?